library/aur.py

```diff
@@ -9,6 +9,7 @@
 import tarfile
 import os
 import os.path
+from distutils.dir_util import copy_tree
```

Comment:
We need a good reason to use this instead of shutil's copytree()

Reply:
What's the exact problem of using copy_tree from distutils instead of copytree from shutil?

Reply:
This is not the standard package for file operations, not sure how well this function will be maintained and why we want to rely on a distribution package